cetmix_tower_aws/readme/CONFIGURE.md

```
@@ -5,7 +5,7 @@
 
    To use the AWS integration with Cetmix Tower, you need to create AWS access keys:
    
-   - Follow the [official AWS documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/security-creds.html) for creating IAM access keys
+   - Follow the official AWS documentation https://docs.aws.amazon.com/IAM/latest/UserGuide/security-creds.html for creating IAM access keys
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)